client/apollo/react/src/Form/ItemMessage/ItemMessageCommon.tsx

```diff
@@ -3,23 +3,29 @@ import successIcon from "@material-symbols/svg-400/outlined/check_circle-fill.sv
 import { Svg } from "../../Svg/Svg";
 
 type ItemMessageProps = {
-  message: string;
+  message?: string;
   id?: string;
-  type?: "error" | "success";
+  messageType?: "error" | "success";
 };
 
 export const ItemMessage = ({
   message,
   id,
-  type = "error",
-}: ItemMessageProps) => (
-  <div className={`af-item-message af-item-message--${type}`}>
-    <Svg
-      src={type === "error" ? errorIcon : successIcon}
-      className="af-item-message__icon"
-    />
-    <span id={id} aria-live="assertive" className="af-item-message__message">
-      {message}
-    </span>
-  </div>
-);
+  messageType = "error",
+}: ItemMessageProps) => {
+  if (!message) {
+    return null;
+  }
+
+  return (
+    <div className={`af-item-message af-item-message--${messageType}`}>
+      <Svg
+        src={messageType === "error" ? errorIcon : successIcon}
+        className="af-item-message__icon"
+      />
+      <span id={id} aria-live="assertive" className="af-item-message__message">
+        {message}
+      </span>
+    </div>
```

Comment:
on pourrait améliorer l'accessibilité en utilisant un `small` au lieu de la `div `et mettre le `svg` en `arria-hidden `non ?
```suggestion
    <small className={`af-item-message af-item-message--${messageType}`}>
      <Svg
        src={messageType === "error" ? errorIcon : successIcon}
        className="af-item-message__icon"
        aria-hidden
      />
      <span id={id} aria-live="assertive" className="af-item-message__message">
        {message}
      </span>
    </small>
```

Reply:
> on pourrait améliorer l'accessibilité en utilisant un `small` au lieu de la `div `et mettre le `svg` en `arria-hidden `non ?

on peut en effet améliorer l'accessibilité mais je pense qu'il faudrait plutot mettre le **role alert** plutot que d'utiliser la balise small : 

```html
 <div
      className={`af-item-message af-item-message--${messageType}`}
      role="alert"
      aria-live="assertive"
    >
      <Svg
      src={messageType === "error" ? errorIcon : successIcon}
      className="af-item-message__icon"
      aria-hidden="true"
      />
      <span id={id} className="af-item-message__message">
      {message}
      </span> 
    </div>
```

Reply:
C'est pas bête mais du coup il faudrait une combinaison des 2.
avoir un small au lieu de la small car dans le cas d'une description du champ cela correspond.
et lors d'une erreur ajouter le role alert pour appuyer qu'il y a quelque chose qui c'est mal passé.

qu'en penses tu ?